components/atoms/Stages.js

```diff
@@ -43,82 +43,84 @@ export default function Stages(props) {
   }, [props.journeys]);
 
   return (
-    <div className="p-4">
-      <div className="w-full flex flex-col sm:flex-wrap sm:flex-row justify-left">
-        <div>
-          <label className="text-custom-blue-reportButton pr-4" htmlFor="stage">
-            {props.labelText}
-          </label>
+    <div className="w-full flex flex-col mb-4 sm:flex-wrap sm:flex-row justify-left">
+      <div>
+        <h4>{props.stagesSubtitle}</h4>
 
-          <select
-            className="w-auto mb-2 px-2 h-8 rounded border bg-white border-gray-400 mt-2 overflow-ellipsis xxs:max-w-full focus:shadow-blue focus:border-custom-blue-focus focus-visible:outline-none"
-            id="stage"
-            onChange={onChangeHandler}
-          >
-            <option key="0" value={0} defaultValue disabled selected>
-              {props.selectPlaceholder}
+        <ul className="text-xs mt-2 mb-6">
+          {props.journeys.map((journey, idx) => (
+            <li key={idx} className="py-2 pl-2 ">
+              <span>
+                <FontAwesomeIcon
+                  icon={faCircle}
+                  className="text-custom-blue-reportButton"
+                />
+              </span>
+              <span className="pl-3">{journey.title}</span>
+            </li>
+          ))}
+        </ul>
+
+        <h4>{props.stagesSelectTitle}</h4>
+
+        <label className="text-custom-blue-reportButton pr-4" htmlFor="stage">
+          {props.stagesSelectLabel}
+        </label>
+
+        <select
+          className="w-auto mb-2 px-2 h-8 rounded border bg-white border-gray-400 mt-2 overflow-ellipsis xxs:max-w-full focus:shadow-blue focus:border-custom-blue-focus focus-visible:outline-none"
+          id="stage"
+          onChange={onChangeHandler}
+        >
+          <option key="0" value={0} defaultValue disabled>
```

Comment:
```suggestion
          <option key="0" value={0} defaultValue disabled selected>
```
Should fix the issue I mentioned above

Reply:
I'll added but I think react does not likes it.